File: AppjedrezApi/AuthApi/Models/Torneo.cs
using System;
using System.Collections.Generic;

namespace AuthApi.Models;

public partial class Torneo
{
    public Guid Id { get; set; }

    public string Nombre { get; set; }

    public DateTime? FechaInicio { get; set; }

    public string Descripcion { get; set; }

    public string Localidad { get; set; }

    public int? IdTipoTorneo { get; set; }

    public int? CantidadParticipantes { get; set; }

    public Guid? IdOrganizador { get; set; }

    public int? EloMinimo { get; set; }

    public int? EloMaximo { get; set; }

    public TimeSpan? HoraInicio { get; set; }

    public bool? Borrado { get; set; }

    public byte[] Portada { get; set; }

    public DateTime? FechaFinal { get; set; }

    public int? EloPromedio { get; set; }

    public string Estado { get; set; }

    public virtual Usuario IdOrganizadorNavigation { get; set; }

    public virtual TipoTorneo IdTipoTorneoNavigation { get; set; }

    public virtual ICollection<Inscripcion> Inscripcions { get; } = new List<Inscripcion>();

    public virtual ICollection<Rondum> Ronda { get; } = new List<Rondum>();
}

File: appjedrez/src/app/models/torneo.ts
import { Inscripcion } from './inscripcion';
import { Ronda } from './ronda';
import { TipoTorneo } from './tipo-torneo';
import { Usuario } from './usuario';

export interface Torneo {
  id: string;
  nombre: string;
  fechaInicio: string;
  fechaFinal: string;
  descripcion: string;
  localidad: string;
  idTipoTorneo: number;
  cantidadParticipantes: number;
  idOrganizador: string;
  eloMinimo: number;
  eloMaximo: number;
  horaInicio: string;
  borrado: boolean;
  estado: string;
  portada: File;
  idOrganizadorNavigation: Usuario;
  tipoTorneo: TipoTorneo;
  inscripciones: Inscripcion[];
  rondas: Ronda[];
}
